tests/webui_eslint/run_webui_eslint.sh

```diff
@@ -27,5 +27,5 @@ npm install
 echo
 
 echo "Linting:"
-npx eslint --format stylish --no-color src/
+npx eslint --format stylish --no-color --ext .js --ext .jsx src/
```

Comment:
Maybe just run the script from package.json? 

`npm run eslint`

Reply:
I think that would become `npm run eslint src/ -- --format stylish --no-color` then...?

Maybe we want to drop one of these two :)